plugins/frontend/backstage-plugin-argo-cd/src/components/ArgoCDDetailsCard.tsx

```diff
@@ -50,17 +50,35 @@ interface Condition {
   type: string;
 }
 
+interface ConditionRow extends Condition {
+  key: string;
+}
+
 const MessageComponent = ({
   conditions,
 }: {
   conditions: Condition[] | undefined;
 }) => {
+  const [mapped, setMapped] = useState<ConditionRow[]>([]);
+
+  useEffect(() => {
+    setMapped(
+      conditions?.map((condition: Condition) => {
+        console.log(`${condition.lastTransitionTime}-${condition.type}`);
```

Comment:
Could we take out this log statement please?

Reply:
Ahh my bad, I thought that I removed all of the `console.logs` :sweat_smile:

Reply:
yeah I do that all the time 😄